.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: main
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - run: npm run build
+      - id: package-version
+        uses: martinbeentjes/npm-get-version-action@v1.1.0
+      - uses: josephbmanley/butler-publish-itchio-action@v1.0.2
+        env:
+          BUTLER_CREDENTIALS: ${{ secrets.BUTLER_CREDENTIALS }}
```

Comment:
this is a secret which will need to be configured under https://github.com/Ayolland/borksy/settings/secrets/actions > "New repository secret"; the name should be `BUTLER_CREDENTIALS`, and the value should be the key from the itch account, which you can create/find under https://itch.io/user/settings/api-keys